client/windows/CAdvmapInterface.cpp

```diff
@@ -956,11 +981,18 @@ void CAdvMapInt::show(SDL_Surface * to)
 
 		if(scrollingDir)
 		{
+			SetScrollingCursor(scrollingDir);
+			ScrollingState = true;

```

Comment:
I think all codebase have more or less consistent style for variables and function names. So please rebase it without capital letters in the beginning.